docs/understanding-radicle/glossary.md

```diff
@@ -21,162 +23,148 @@ changes committed to a [project][pr] history along with who made them and when.
-
-## project metadata
-The [project][pr] metadata is stored in [project][pr].json. It includes the
-name, description, and maintainer identities (i.e. PeerIDs) of the
-[project][pr]. Any update to the [project][pr] metadata must be signed by a
-quorum (as defined by the [project][pr]'s governance) of maintainers. The
-history of the metadata file is backed by the underlying DVCS (usually Git),
-and is verified upon replication.
+carries a unique, shareable [Radicle ID](Radicle ID). A [project][pr] also
+includes the identities of all its [maintainers](maintainers) The entirety of
+the [project][pr] data and metadata, including social artifacts such as
+comments, are stored within the [project][pr]'s [repository][re]. [project][pr]s
+are the principle unit of replication.
 
 ## project name
-A human-readable [project][pr] name that is given to the [project][pr] ID. It
-is not guaranteed to be unique
+A human-readable [project][pr] name that is chosen for a project. It is not
+guaranteed to be unique.
 
 ## publish
 To make data public to the network. Once something is published, it may be
-replicated and stored in multiple copies on the network.
+fetched and replicated by connected peers.
 
-## rad remote
-The remote a [user][us] has to push to in order for their [repository][re] to
-be published on Radicle. This is the only remote managed by Radicle that can be
-pushed to.
-
-`git push rad`
+## Radicle ID
+Unique shareable identifiers for peers and projects in the Radicle Network. 
 
 ## Radicle Link
-A peer-to-peer replication protocol built on Git. You can read more about it
-[here][rl].
+A peer-to-peer replication protocol built on Git. See [How it
+Works](how-it-works.md).
 
 ## Radicle network
-The network of peers that are replicating on the Radicle Link protocol.
+The network of peers that replicate and gossip data with Radicle Link.
 
 ## radicle-upstream
-A desktop application (graphic [user][us] interface, GUI) built to interact
-with and enable access to the Radicle Network and, initially, the primary
-end-[user][us] experience. However, it is one of many potential clients that
-[user][us]s can use to access the Radicle Network.
+A desktop application (graphic [user][us] interface, GUI) built to interact with
+and enable access to the Radicle Network and, initially, the primary
+end-[user][us] experience. However, it will be one of many potential clients
+that [user][us]s can use to access the Radicle Network.
 
 ## remote
-A remote copy of a [repository][re] with the same [project][pr] ID, tracked by
-the VCS. Remotes can be pulled/fetched from. Maintainers of a [repository][re]
-may choose to track the repositories of other owners. If the remote
-[repository][re] (or "remote," in Git terminology) is found to track other
-remotes, the tracking [repository][re] shall also transitively track those, up
-to a configurable N degrees out.
+Another peer's view of a project. Remotes can be pulled/fetched from.
```

Comment:
We use "view" in a few places, I think we should use the terms "tree" or "repository" to be clearer.

Reply:
`tree` is one of those terms which is easily, heavily overloaded in the domain - source code control and sharing - we operate in. It would be helpful to find a more distinct way of referring to the project state other peers maintain. In another place we discussed to settle back on `fork` again, when a user created a distinct version of a project on their device. Maybe that is a term we can establish, as it disassociate from the physicality of a file/folder hierarchy.

Reply:
yeah I agree, I mean especially if we have a "fork" button. It's a bit unfortunate though, because it makes it harder to talk about a fork in the more common sense.

Reply:
Agreed, that's why I proposed `view` initially.

Reply:
So we are keeping `view`? or replacing with `tree`?